src/sentry/options/manager.py

```diff
@@ -14,6 +16,21 @@
 NoneType = type(None)
 
 
+class UpdateChannel(Enum):
+    """Enumerated type referring to potential sources of Sentry Options"""
+
+    UNKNOWN = "unknown"
+    APPLICATION = "application"
+    AUTOMATOR = "automator"
+    CLI = "cli"
+    KILLSWITCH = "killswitch"
+    ADMIN = "admin"
```

Comment:
might be nice to have docstring comments on some of these

Reply:
right, makes sense

Reply:
done